### ISSUE TITLE
Upstream bug: menu appears at wrong location with scaling on

### ISSUE THREAD

**Issue by yuyichao** [Nheko-Reborn/nheko#618]
### Describe the bug
I'm not sure if you tracks upstream bugs here so feel free to close this. The upstream bug report is here https://bugreports.qt.io/browse/QTBUG-94619 . I just want to post it here in case someone else hit the same here and to help searching. It's quite a major usability issue I'd say...

### To Reproduce
See upstream bug report. This affects the right click menu in message view.

### System:

- Nheko version: 
- Profile used: 
- Installation method: 
- Operating System:
- Qt version: 
- C++ compiler: 
- Desktop Environment:

**Comment by deepbluev7** [comment]
I don't know of any workaround for this, so we will probably need to wait for this to be fixed upstream. Thank you for reporting this upstream though!

**Comment by yuyichao** [comment]
Right. I looked through the code and it seems pretty benign which is why I checked if it's just a Qt bug...... as I said, just a heads up...

**Comment by ThisNekoGuy** [comment]
Was this reported **_to_** upstream though?

**Comment by evren320** [comment]
linux version is affected from this bug, but scaling on windows version works normally

**Comment by deepbluev7** [comment]
Fixed in the KDE patch collection for Qt 5.15.7: https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=560d990f7afbe0ee816e0c0d12fd8d07c5efaf70 https://invent.kde.org/qt/qt/qtquickcontrols2/-/merge_requests/8
